client/my-sites/themes/themes-banner/index.jsx

```diff
@@ -92,13 +109,18 @@ const mapStateToProps = ( state, { themeId } ) => {
 	const siteId = getSelectedSiteId( state );
 	const themeUrl = getThemeDetailsUrl( state, themeId, siteId );
 	const activeThemeId = getActiveTheme( state, siteId );
+	const isBannerVisible = isThemesBannerVisible( state );
 	return {
 		siteId,
 		themeUrl,
 		activeThemeId,
+		isBannerVisible,
 	};
 };
 
-export default connect( mapStateToProps, {
+const mapDispatchToProps = dispatch => ( {
 	recordTracksEvent: recordTracksEventAction,
-} )( localize( ThemesBanner ) );
+	onBannerClose: () => dispatch( hideThemesBanner ),
```

Comment:
No need to use the `dispatch =>` longform of `mapDispatchToProps`. Let's continue to use the object shorthand instead. Also, we recommend against naming actions (and even component methods) that we pass as event handlers to start with `on...`: rather, they should describe what they do, so let's continue to pass `hideThemesBanner` to the component. Finally, let's continue to inline `mapDispatchToProps`:

```jsx
import { hideThemesBanner as hideThemesBannerAction } from 'state/themes/themes-ui/actions'

/*...*/

export default connect( mapStateToProps, {
	hideThemesBanner: hideThemesBannerAction,
	recordTracksEvent: recordTracksEventAction,
} )( localize( ThemesBanner ) );
```